### PR TITLE
Add first rewriting patterns for llama adding onnxruntime contrib ops

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

**Review by titaiwangms — COMMENTED** [review]
Currently, we separate ort rewriter to this folder https://github.com/microsoft/onnxscript/blob/main/onnxscript/rewriter/onnxruntime/__init__.py

File: onnxscript/rewriter/onnxruntime/fused_matmul_rule_sets.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        kwargs["alpha"] = alpha.value / c if alpha else 1.0 / c
+        for name in ["transA", "transB", "transBatchA", "transBatchB"]:
+            att = node.attributes.get(name)
+            if att:
+                kwargs[name] = att.value
+        return op.FusedMatMul(x, y, **kwargs, domain="com.microsoft")
+
+
+class _TransposeMatMulBase(orp.RewriteRuleAsClass):
+    _pos: ClassVar = 1
```
Comment (gramalingam) [inline onnxscript/rewriter/onnxruntime/fused_matmul_rule_sets.py:67]: Maybe add a comment documenting this? Something like "Indicates the input of MatMul that is transposed."

Comment (gramalingam) [reply onnxscript/rewriter/onnxruntime/fused_matmul_rule_sets.py:67]: So you have patterns where only one of the two inputs is transposed? Wondering if eventually you would need to support both inputs being transposed.

**Review by gramalingam — APPROVED** [review]

File: onnxscript/rewriter/onnxruntime/fused_matmul_rule_sets.py (hunk 1 above)
Comment (xadupre) [reply onnxscript/rewriter/onnxruntime/fused_matmul_rule_sets.py:67]: I have but I'll add them in another PR.

**Pull request merged**